CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
-## 2.0.1-dev
+## 2.0.1
 
+- Switch to using `package:lints`.
```

Comment:
[nit] This change doesn't impact users. I think we can omit this form the changelog.

Reply:
gotcha, will remove